group_vars/all

```diff
@@ -296,7 +296,8 @@
   lmod_db_passwd: "password"
   lmod_db_allow_host: "localhost"
   lmod_db_host_machine: "ohpc"
-  lmod_slurm_module_rel_path: "slurm/slurm"
+  lmod_slurm_module_name: "slurm/slurm"
```

Comment:
This is causing issue in packer image builds for compute and OOD, because the slurm module name is just `slurm`. We need to find a way to fix it, may be we can do it by `group_vars/prod` ? Any ideas ?

Reply:
Instead of only specifying  slurm module name, we should probably use a list of default modules, and create DefaultModules.lua template with the said variable

Reply:
I agree with the list idea. Also to add on to the `group_vars/prod` idea, I think that should work once I have the pipeline and I clone from the prod(uabrc) CRI and pull in dev branch from there, we would have both prod and dev vars, then the var value would be overridden. 
Currently in my builds I am using dev branch only and the var override is not coming into the picture yet. So I think that is why i am still seeing this issue slurm/slurm is not a module once I spin up the image. Once we go into production this problem wouldn't show up.